gogs_client/entities.py

```diff
@@ -154,6 +170,33 @@ def fork(self):
         return self._fork
 
     @property
+    def default_branch(self):
+        """
+        The name of the default branch
+
+        :rtype: bool
+        """
+        return self._default_branch
+
+    @property
+    def empty(self):
+        """
+        Whether the repository is empty
+
+        :rtype: bool
+        """
+        return self._empty
+
+    @property
+    def size(self):
+        """
+        Size of the repository
```

Comment:
If you happen to know the unit of the `size` field (e.g. kilobytes, megabytes, etc.), please add it to the description